game-core/src/main/java/games/strategy/engine/framework/startup/ui/panels/main/game/selector/GameSelectorPanel.java

```diff
@@ -317,14 +317,11 @@ private void updateGameData() {
           loadSavedGame.setEnabled(canSelectGameData || canChangeHostBotGameData);
           loadNewGame.setEnabled(canSelectGameData || canChangeHostBotGameData);
           // Disable game options if there are none.
-          if (canChangeHostBotGameData
-              || (canSelectGameData
-                  && model.getGameData() != null
-                  && !model.getGameData().getProperties().getEditableProperties().isEmpty())) {
-            mapOptions.setEnabled(true);
-          } else {
-            mapOptions.setEnabled(false);
-          }
+          mapOptions.setEnabled(
```

Comment:
To some extent one could argue each automated update should be applied one by one:
1. Can more easily review as all changes are the same type
2. Commit history documentation is better, can copy/paste description of the change type into the commit summary

There is though a running debate whether automated passes are always the right thing. In this code, there is more work to be done IMO to really fix it up. For example, we have one boolean condition extracted to a variable and a second boolean condition that is fully written out. Overall, it's hard to grok. One could argue that if you're going to touch this up, then this location should be fully fixed up to avoid churn.

While I support mass cleaning, there are some aspects of it that do have me on the fence.